modules/programs/zsh.nix

```diff
@@ -220,6 +220,21 @@ let
     };
   };
 
+  autosuggestionModule = types.submodule {
+
+    options = {
+        enable = mkOption {
+          default = false;
+          description = "Enable zsh autosuggestions";
+        };
+        highlight = mkOption {
+          default = "fg=cyan";
+          description = "Enable zsh autosuggestions";
```

Comment:
I imagine that the description here needs to be updated to match the option name.